packages/react-button/src/components/ToggleButton/checkedBehaviorDefinition.ts

```diff
@@ -0,0 +1,30 @@
+import { BehaviorRule, Rule } from '@fluentui/a11y-testing';
+
+export const checkedBehaviorDefinition: Rule[] = [
+  BehaviorRule.root()
+    .doesNotHaveAttribute('aria-pressed')
+    .description('by default'),
+  BehaviorRule.root()
+    .afterEvent(
+      'onClick',
+      new MouseEvent('click', {
+        view: window,
+        bubbles: true,
+        cancelable: true,
+      }),
+    )
+    .hasAttribute('aria-pressed', 'true')
+    .description('after the element has been clicked'),
+  BehaviorRule.root()
+    .forProps({ checked: undefined, defaultChecked: true })
+    .afterEvent(
+      'onClick',
```

Comment:
`"onClick"` as name of the handler is actually required only for the hooks tests. If we decide to go with only components test, this can safely be removed, as in the `ComponentTestFacade` we are just dispatching the event which is currently second argument.